### ISSUE TITLE
Mobile fly-out nav should be in tab order, accept Esc to dismiss

### ISSUE THREAD

**Issue by jeremy-w** [jeremy-w/orthic#22]
It should also be in the tab order. It does present as a menu when focused using Voice Over, but I cannot access it using the keyboard.

And possibly there are other considerations. See https://www.w3.org/WAI/tutorials/menus/flyout/.
